.github/workflows/pylint_checking.yml

```diff
@@ -0,0 +1,44 @@
+---
+name: Pylint
+on: pull_request
+jobs:
+  formatting:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.9]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Python ${{ matrix.python-version }} for Pylint
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install Pylint
+        run: pip install pylint
+      - name: Check Score
+        run: |
+          echo "Start checking scores"
+          pylint `find . -name "*.py"` | grep 'Your code has been rated' | awk '{print $10}' | awk -F/ '{print $1}'
```

Comment:
does this pylint all py files in the repo or just the one within the coast directory?

Reply:
The pylint will run in all python files, including _init.py, config.py, tests, setup.py. It is normally the way I apply Pylint on my codes. However, I think we can change it to just the coast directory.

Reply:
@thogar-computer , if you prefer, I can update the pull request with the pylint only for the coast directory

Reply:
@soutobias thats ok. this can be closed